docs/summaries.md

```diff
@@ -0,0 +1,222 @@
+# TensorBoard: Visualizing Learning
+
+The computations you'll use TensorFlow for - like training a massive
+deep neural network - can be complex and confusing. To make it easier to
+understand, debug, and optimize TensorFlow programs, we've included a suite of
+visualization tools called TensorBoard. You can use TensorBoard to visualize
+your TensorFlow graph, plot quantitative metrics about the execution of your
+graph, and show additional data like images that pass through it. When
+TensorBoard is fully configured, it looks like this:
+
+![MNIST TensorBoard](https://www.tensorflow.org/images/mnist_tensorboard.png "MNIST TensorBoard")
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="eBbEDRsCmv4"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+This 30-minute tutorial is intended to get you started with simple TensorBoard
+usage. It assumes a basic understanding of TensorFlow.
+
+There are other resources available as well! The [TensorBoard GitHub](https://github.com/tensorflow/tensorboard)
+has a lot more information on using individual dashboards within TensorBoard
+including tips & tricks and debugging information.
+
+## Setup
+
+[Install TensorFlow](../install/). Installing TensorFlow
```

Comment:
For API symbols written in backticks (\`tf.summary\`), this are automatically expanded to the correct link on import.

Since you're now in a *subsite* and outside the [tensorflow/docs repo guides](https://github.com/tensorflow/docs/tree/master/site/en), this won't work when browsing the markdown in your repo. When it is imported into tensorflow.org/tensorboard, this relative link will work.

If you want this to be browseable in github, you would have to include the full path like:
\[Install TensorFlow](https://www.tensorflow.org/install/)

Also, for relative links within your repo, include the file extension like:
\[a tutorial](./tutorials/tut.ipynb) ---this way it's browseable in github, but that extension is stripped when imported into the website and it will link to the html page.

Ultimately, we optimize for making the links on tensorflow.org work, but browseable in GitHub is a nice to have.

Reply:
Great; thanks for the explanation. These are good to know.